src/core/ext/filters/client_channel/http_proxy_mapper.h

```diff
@@ -35,14 +35,13 @@ namespace grpc_core {
 
 class HttpProxyMapper : public ProxyMapperInterface {
  public:
+  static constexpr char const* ADDRESS_PROXY_ENV_VAR = "grpc_address_proxy";
```

Comment:
This should be named `kAddressProxyEnvVar`, as per:

https://google.github.io/styleguide/cppguide.html#Constant_Names